+adi/+AD2S1210/Rx.m

```diff
@@ -0,0 +1,123 @@
+classdef Rx < adi.common.Rx & matlabshared.libiio.base & adi.common.Attribute
+    % AD2S1210 Resolver-to-Digital Converter Class
+    % adi.AD2S1210.Rx Receives data from the AD2S1210 Resolver
+    %   The adi.AD2S1210.Rx System object is a signal source that can receive
+    %   data from the AD2S1210.
+    %
+    %   rx = adi.AD2S1210.Rx;
+    %   rx = adi.AD2S1210.Rx('uri','192.168.2.1');
+    %
+    %   <a href="https://www.analog.com/media/en/technical-documentation/data-sheets/ad2s1210.pdf">AD2S1210 Datasheet</a>
+
+    properties (Nontunable)
+        % SamplesPerFrame Samples Per Frame
+        %   Number of samples per frame, specified as an even positive
+        %   integer.
+        SamplesPerFrame = 4096
+    end
+
+    properties
+        % Angle
+        %   Resolver angle in Degrees.
+        Angle = 0
+
+        % AngleScale Angular Scale
+        %   Resolver angle scale.
+        AngleScale = 0
+
+        % Velocity Angular Velocity
+        %   Resolver velocity in revolutions per second.
+        Velocity = 0
+
+        % VelocityScale Velocity Angular Scale
+        %   Resolver velocityscale.
+        VelocityScale = 0
+
+        % ExcitationFrequency Excitation Frequency
+        %   Resolver excitation frequency in Hertz.
+        ExcitationFrequency = 0
+    end
```

Comment:
Anything with a get. method should be a Dependent property https://www.mathworks.com/help/matlab/matlab_oop/access-methods-for-dependent-properties.html

Reply:
Done.